pkg/autoscaler/autoscaler/autoscaler_manager.go

```diff
@@ -205,3 +158,45 @@ func (am *autoScalerManager) gracefullyDeleteTidbCluster(deleteTc *v1alpha1.Tidb
 
 	return am.deps.Clientset.PingcapV1alpha1().TidbClusters(deleteTc.Namespace).Delete(deleteTc.Name, nil)
 }
+
+func (am *autoScalerManager) updateLastSyncingTimestamp(tac *v1alpha1.TidbClusterAutoScaler, memberType string, group string) error {
```

Comment:
We should update tac once during each sync, just update the status during the sync function and then update the tac, no need to update it for each group status change.

Reply:
Updated